src/views/components/common/PopupCourseBlock.tsx

```diff
@@ -37,6 +39,8 @@ export default function PopupCourseBlock({
     dragHandleProps,
 }: PopupCourseBlockProps): JSX.Element {
     const [enableCourseStatusChips, setEnableCourseStatusChips] = useState<boolean>(false);
+    const [isCopied, setIsCopied] = useState(false);
```

Comment:
stay consistent with the other uses of useState

```suggestion
    const [isCopied, setIsCopied] = useState<boolean>(false);
```